noxfile.py

```diff
@@ -4,16 +4,16 @@
 @nox.session(python=["3.6", "3.7", "3.8", "3.9", "3.10", "3.11", "pypy3"])
 def tests(session):
     session.install(".[test]")
-    session.run("coverage", "run", "-m", "pytest", "-v")
+    session.run("coverage", "run", "-m", "pytest", "-v", *session.posargs)
 
 
-@nox.session
+@nox.session(reuse_venv=True)
```

Comment:
This can be enabled by using `-r` flag, I think I'd prefer to not reuse venv by default, and only do that when the switch is on.

Reply:
The idea is that these don't install the package, so reinstalling the environment shouldn't be needed, so it's a better default. The test environment still defaults to reinstalling the environment. But sure, I can.